tests/features/tags/add-tag.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { expect } from "@playwright/test";
+import { test } from '../../config/test-base';
+import { go_to_page } from "../../ui/steps/login";
+
+test.describe('Tags', function () {
+    let tags_all;
+    let tag_name;
+
+    test('Add new Tag', async ({steps: {pages}, page}) => {
+        tag_name = 'Test_tag_name';
+
+        await test.step('I open Tags page', async () => {
+            await go_to_page(page, 'http://localhost:8080/');
```

Comment:
I propose to define `baseURL` in PW configs.

After that we will be able to use short URL (instead of full path) when navigate to any page.

More info: https://playwright.dev/docs/api/class-testoptions#test-options-base-url

Reply:
done